repository-data/application/src/main/resources/hcm-actions.yaml

```diff
@@ -16,4 +16,6 @@ action-lists:
       /content/documents/administration/valuelists/documenttypes: reload
       /content/documents/administration/valuelists/kls: reload
       /content/documents/administration/valuelists/kls/blogcategories: reload
-      /content/documents/administration/valuelists/kls/bulletincategories: reload
\ No newline at end of file
+      /content/documents/administration/valuelists/kls/bulletincategories: reload
+  - 1.0.9:
+      /content/documents/administration/valuelists/listingtypes: reload
```

Comment:
If this `listingtypes' is a fresh new valuelist document then we may not need to add action to reload it ?

Reply:
@thojava - That's true. Ideally we do not need to add new documents. But, I have observed on my local instance that the reload of `/content/documents/administration/valuelists` deletes the node, but recreates only that node and not its children and so I have added `/content/documents/administration/valuelists/listingtypes: reload` to ensure that it is added on prod.
```
  - 1.0.8:
      /content/documents/administration/valuelists: reload
```
I am happy to remove this if you think it will definitely be added without the above action. Let me know

Reply:
@smanivasag I see your points. I think logically we should not have this reload 
`/content/documents/administration/valuelists/listingtypes: reload`. Should just try to remove it to see how it working on PROD deployment, it is small thing anyway